indigo_app/js/components/TaxonomyTOC.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="card col-md-3 p-3 mx-1 mb-1">
+    <la-table-of-contents-controller
+        :items.prop="taxonomy_toc"
+        collapse-all-btn-classes="btn btn-sm btn-secondary"
+        expand-all-btn-classes="btn btn-sm btn-secondary"
+        title-filter-clear-btn-classes="btn btn-sm btn-secondary"
+        title-filter-input-classes="form-field"
+        :class="active"
+    ></la-table-of-contents-controller>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TaxonomyTOC',
+  data () {
+    return {
+      taxonomy_toc: JSON.parse(document.querySelector('#taxonomy_toc').textContent),
+    };
+  },
+  mounted () {
+    const params = new URLSearchParams(window.location.search);
+    const toc = document.getElementsByTagName('la-table-of-contents-controller');
+    toc[0].addEventListener('itemRendered', (e) => {
+      const tocItem = e.target;
+      if (!tocItem) return;
+      const anchor = tocItem.querySelector('.content__action__title');
+      const href = new URLSearchParams(anchor.getAttribute('href'));
+      if (params.get('taxonomy_topic') === href.get('taxonomy_topic')) {
+        anchor.classList.add('active');
+      }
+    });
+  }
+};
+</script>
+
+<style>
+.active, .active:hover {
+  color: tomato;
```

Comment:
In the document view we apply this styling, which users are used to:

```css
  background-color: #2d7ad4;
  color: white;
```

Reply:
Changed :heavy_check_mark: